src/geo/leaflet/leaflet-layer-view.js

```diff
@@ -1,15 +1,18 @@
 var _ = require('underscore');
 var Backbone = require('backbone');
 
-var LeafletLayerView = function (layerModel, leafletMap) {
+var LeafletLayerView = function (layerModel, leafletMap, mapModel) {
```

Comment:
This is the base class for all layer "views" (`LeafletTiledLayerView`, `LeafletCartoDBLayerView`, etc...). Do we want to track errors in all those types of views? Or... do we only want to check errors in the case of a `LeafletCartoDBLayerView` which is the one that will go against the Maps API? We could push this down to `src/geo/leaflet/leaflet-cartodb-layer-group-view.js` in that case.

(this would also apply to `src/geo/gmaps/gmaps-layer-view.js`).

Reply:
I put it there because I wasn't sure which ones will track errors, if I'm not mistaken in the case of leaflet there are two views right? `LeafletCartoDBLayerGroupView ` and `LeafletTiledLayerView `

Reply:
[There are more types](https://github.com/CartoDB/cartodb.js/tree/v4/src/geo/leaflet)... many of those won't trigger the `tileerror`. For example, we use a `LeafletTiledLayerView` to render the basemap (from CARTO, or other providers). I don't think we want to check tiles in that case because CARTO might not be serving those tiles...

Reply:
So, we should just check `LeafletCartoDBLayerGroupView` and `GMapsCartoDBLayerGroupView`, right?

Reply:
I think so, yes.

Reply:
👍